test/RootStateSchema.json

```diff
@@ -710,6 +742,14 @@
             ],
             "type": "string"
         },
+        "FiatAccountType": {
+            "enum": [
+                "BankAccount",
```

Comment:
just for my understanding, if there are more values for this enum in future, would it require a schema migration?

Reply:
I am not sure if this would need a schema migration. I'm not sure what we'd "migrate" if more enums were added. @jeanregisser do you know?